src/rtctools/optimization/optimization_problem.py

```diff
@@ -221,7 +225,14 @@ def optimize(
     def __check_bounds_control_input(self) -> None:
         # Checks if at the control inputs have bounds, log warning when a control input is not
         # bounded.
-        bounds = self.bounds()
+        if self.ensemble_specific_bounds:
+            # We can just naively merge dictionaries, as we're just checking for
+            # presence of keys.
```

Comment:
What if bounds are missing for a control variable in one of the members? Wouldn't we want to raise the warning in that case?